components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -28,6 +28,16 @@ import (
 	"github.com/chef/automate/lib/io/fileutils"
 )
 
+const restoreRecovery = `Check the logs (journalctl -f -u chef-automate) for errors related
```

Comment:
@chef/automate-deployment-team I could use input/feedback on what it is we want to say in the recovery message. This would have gotten the customer who filed the original ticket to a place where they could restore, but I'm not sure about it as general advice.

Reply:
I think the only thing that might be confusing is the `-f`. If we're following we'll probably only see new log lines and not what potentially went wrong.